src/components/ContactCard/ContactAccounts.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import { Text, Caption } from 'cozy-ui/transpiled/react/Text'
+import Icon from 'cozy-ui/react/Icon'
+
+import IconGoogle from '../../assets/icons/connect-google.svg'
+
+const ACCOUNTS_MAPPING = {
+  'konnector-google': {
+    label: 'Google',
+    icon: IconGoogle
+  }
+}
+
+const ContactAccounts = ({ accounts, t }) => (
+  <div className="contact-accounts">
+    <h3 className="contact-fields-title">
+      {t('associated_accounts', {
+        smart_count: accounts.length
+      })}
+    </h3>
+    <ol className="contact-field-list">
+      {accounts.map(account => {
+        const { icon, label } = ACCOUNTS_MAPPING[account.type]
+        return (
+          <li key={account.id}>
+            <div className="contact-field">
+              <div className="contact-field-icon">
+                <Icon icon={icon} height="16" width="16" />
```

Comment:
I think 16 is already the default size (https://github.com/cozy/cozy-ui/blob/master/react/Icon/index.jsx#L6)